test/system/randomwalk/conf/modules/MultiTable.xml

```diff
@@ -29,11 +29,11 @@
 
 <node id="dummy.ToAll">
   <edge id="mt.CreateTable" weight="20"/>
-  <edge id="mt.Write" weight="100"/>
+  <edge id="mt.Write" weight="10"/>
   <edge id="mt.CopyTable" weight="5"/>
+  <edge id="mt.BulkImport" weight="100"/>
```

Comment:
Do these relative weights make sense as default or were they just what you were using to test?

Reply:
I think so. I modified the defaults to be more bulk import heavy, since they seem to be more popular with users than strictly batch writing. The more bulk imports make the cluster take longer to run MajC. With only the batchWriter test, flushes were happening a lot and MajC were not taking that long.